libs/npm/core/package.json

```diff
@@ -49,7 +49,6 @@
     "react-tooltip": "^5.10.0",
     "styled-components": "^6.1.11",
     "stylis": "^4.3.2",
-    "tno-core": "0.1.121",
```

Comment:
I don't understand why this keep slipping into this dependency list.  It's very annoying.